content/en.json

```diff
@@ -143,16 +136,16 @@
       ]
     },
     "technology": {
-      "title": "Used Technology",
+      "title": "Technology",
       "subtitle": "We prefer using \"<em>the best tool for the job</em>\"; we're generalists who can quickly specialize when the need arises.",
       "how-title": "How we like to do this:",
       "how": [
-        "We do custom-made software; aiding in buy-over-build where needed",
-        "Hosting (usually) in the cloud",
+        "OS is a personal choice per developer",
+        "We make custom software and aid in buy-over-build",
+        "Hosting in the cloud",
         "Back-end in a modern stack",
-        "Front-end normally with mature frameworks",
-        "Mobile cross-platform, unless \"native\" <em>really</em> adds value",
-        "Data storage fitting for the project"
+        "Front-end with mature frameworks",
+        "Mobile, cross-platform applications"
```

Comment:
This looses information I think? Although we'd often go for cross-platform solutions, we would (and have in the past) assist with Native solutions when it makes sense. I'd prefer to see the "native" part still, though we may be able to make it more brief nonetheless?

Reply:
Added back the "native" part